ocs_ci/ocs/cluster.py

```diff
@@ -995,24 +995,30 @@ def validate_pg_balancer():
 
     """
     # Check OSD utilization either pg balancer is active
+    # TODO: Revisit this if pg difference value needs change
+    # TODO: Revisit eval value if pg balancer mode changes from 'upmap'
     if get_pg_balancer_status():
         eval = get_balancer_eval()
         osd_dict = get_pgs_per_osd()
-        osd_min_pg_value = min(osd_dict.values())
-        osd_max_pg_value = max(osd_dict.values())
-        diff = osd_max_pg_value - osd_min_pg_value
-        # TODO: Revisit this if pg difference value needs change
-        # TODO: Revisit eval value if pg balancer mode changes from 'upmap'
-        if diff <= 5 and eval <= 0.02:
+        osd_avg_pg_value = round(sum(osd_dict.values()) / len(osd_dict))
+        osd_pg_value_flag = True
+        for key, value in osd_dict.items():
+            diff = abs(value - osd_avg_pg_value)
+            if diff <= 10:
```

Comment:
@ramkiperiy You may need to change to docstring to match this new change.

Reply:
@keesturam updated doc string accordingly, please review and approve